includes/Notifications/EchoRequestMoreDetailsPresentationModel.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Miraheze\CreateWiki\Notifications;
+
+use EchoDiscussionParser;
+use EchoEventPresentationModel;
+use RawMessage;
+
+class EchoRequestDeclinedPresentationModel extends EchoEventPresentationModel {
```

Comment:
i18 for the new messages needs to be introduced in en.json

Reply:
Also, change the class name to `EchoRequestMoreDetailsPresentationModel.php`

Reply:
Class is already declared elsewhere, needs to be renamed.